pubsub/runtime/src/main/java/io/quarkiverse/googlecloudservices/pubsub/QuarkusPubSub.java

```diff
@@ -46,13 +45,6 @@ void init() {
         }
     }
 
-    @PreDestroy
-    void destroy() throws Exception {
-        if (channelProvider.isPresent()) {
-            channelProvider.get().getTransportChannel().close();
-        }
-    }
```

Comment:
Why do you remove it?
As far as I rember, without this, ITs shows error messages...

Reply:
I'll have a look at it, but the main reason is that this causes the slow down in the main IT we see. Basically, that channel provider and the transport channel is owned by the pubsub logic itself, so we shouldn't close it manually. It will get automatically closed at the appropriate time by the pubsub logic.

Reply:
OK then +1